service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyUpgradeIT.java

```diff
@@ -31,18 +36,14 @@
 import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
 import org.kaazing.k3po.junit.annotation.Specification;
 import org.kaazing.k3po.junit.rules.K3poRule;
-
-import java.net.URI;
-
-import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.junit.rules.RuleChain.outerRule;
+import org.kaazing.test.util.MethodExecutionTrace;
 
 public class HttpProxyUpgradeIT {
 
-    @Rule
     public TestRule timeout = new DisableOnDebug(new Timeout(10, SECONDS));
```

Comment:
You may modify these be to private final (as they don't have annotations to be processed by junit)